glutton.go

```diff
@@ -3,24 +3,52 @@ package glutton
 import (
 	. "fmt"
 	"github.com/hectane/go-nonblockingchan"
+	"gopkg.in/yaml.v2"
 	"honnef.co/go/netdb"
+	"io/ioutil"
 	"os"
+	"path/filepath"
 	"strconv"
 	"time"
 )
 
-// getProtocol(80, "tcp")
-func GetProtocol(port int, transport string) *netdb.Servent {
-	prot := netdb.GetProtoByName(transport)
-	return netdb.GetServByPort(port, prot)
+// For the fields of services.conf
+type Config struct {
+	Description string
+	Ports       map[int]string
 }
 
-// CheckError handles errors
-func CheckError(err error) {
+var ser Config
```

Comment:
Every new connection in glutton calls `getHost()` and it return the address of  remote services on the base of `ser` variable. So I load `services.yml` in global variable `ser` at the startup of program to save the operation ons every call.